odoo_backup_sh/controllers/main.py

```diff
@@ -48,26 +55,31 @@ def _render_template(self, **d):
 
 class BackupController(http.Controller):
 
-    @http.route('/web/database/backups', type='http', auth="none")
-    def backup_list(self):
-        config._parse_config()
-        user_key = config.get('odoo_backup_user_key')
+    def backup_service_request(self, redirect=None):
```

Comment:
``load_backup_list``?

Reply:
ok, but you have taken into account that the function will return auth_link if the user is not logged on odoo-backup.sh

Reply:
``load_backup_list_from_service`` + *note in method description about auth link*?